### PR TITLE
Improved plugin

### PR DESCRIPTION
**Pull request by fabien**
Added support for callbacks, recent Redactor icon class use and selection restore on cancel.

### CODE REVIEW

**Comment by dmitry-mukhin** [comment]
@fabien can you add a changelog record and add yourself to authors list?
Thanks!

**Comment by fabien** [comment]
@dmitry-mukhin just updated the version number and changelog

File: changelog.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - compatibility with latest Redactor 2 toolbar icons (default)
 - new setting: buttonIcon - use 'fa fa-picture-o' for backwards compatibility
 - restore editor selection on cancel
+- assign data-image and data-file attributes for Redactor storage management
```
Comment (dmitry-mukhin) [inline changelog.md]: can you give a link to redactor docs on that feature?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "uploadcare-redactor",
-  "version": "1.5.2",
+  "version": "1.5.3",
```
Comment (dmitry-mukhin) [inline package.json]: these changes call for a minor version change (1.6.0) :)

**Comment by dmitry-mukhin** [comment]
awesome, thanks!

**Pull request merged**